docs-ref-services/sql.md

```diff
@@ -19,46 +19,67 @@ ms.service:
 
 Work with data stored in  [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/sql-database-technical-overview) from Python with the Microsoft ODBC driver. 
 
-The management libraries provide an interface to create, manage, and scale Azure SQL Database deployments from your Python code. Set up and manage databases in [elastic pools](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool) to share resources and configure databases across multiple regions from your code.
+## Client ODBC driver and pyodbc
+Connect to the Azure SQL Database from your applications using the [Microsoft ODBC driver and pyodbc](https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python#install-the-python-and-database-communication-libraries).
 
-## Install the libraries
+### Example
 
-### ODBC driver 
-[Instructions](https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python#install-the-python-and-database-communication-libraries) for downloading the Microsoft ODBC driver.
+Connect to a SQL database and select all records in a table.
 
-### Management
+```python
+connection_string = 'Driver={ODBC Driver 13 for SQL Server};Server=tcp:samplesqlserver123.database.windows.net,1433;Database=samplesqldb;Uid=mysecretname@samplesqlserver123;Pwd={testpassword};Encrypt=yes;Timeout=30;'
```

Comment:
The connection string is brutal. In our notebooks in notebooks.azure.com, we document:

```python
import pyodbc

server = 'yourserver.database.windows.net'
database = 'yourdatabase'
username = 'yourusername'
password = 'yourpassword'

driver= '{ODBC Driver 13 for SQL Server}'
cnxn = pyodbc.connect('DRIVER='+driver+';PORT=1433;SERVER='+server+';PORT=1443;DATABASE='+database+';UID='+username+';PWD='+ password)
cursor = cnxn.cursor()
```